docs/evm-sidechain/axelar-interchain-evm-sidechain-xrpl.md

```diff
@@ -0,0 +1,40 @@
+---
+html: axelar-interchain-transfer-evm-sidechain-xrpl.html
+blurb: Axelar interchain transfer to XRPL.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# Axelar Interchain Transfer from XRPL
+
+## Using Portal
```

Comment:
Remove this section header, unless there are plans for alternative bridging methods.